source/manual/world-taxonomy.html.md

```diff
@@ -0,0 +1,90 @@
+
+World taxons are rendered differently on the frontend to topic taxons. Compare https://www.gov.uk/education/running-and-managing-a-school with https://www.gov.uk/world/yemen.
+
+[topic-taxonomy]: https://docs.publishing.service.gov.uk/manual/taxonomy.html
+[content-api]: https://www.gov.uk/api/content
+[world-all]: https://www.gov.uk/api/content/world/all
+[spec-orgs]: https://github.com/alphagov/whitehall/blob/56006c6f6ba033fbe450ef91d46204499e62e337/config/worldwide_tagging_organisations.yml
+[specific-doc-types]: https://github.com/alphagov/whitehall/blob/8f1b71d7faa130547a3fca621542b6f9f865034b/app/models/edition/taggable_organisations.rb#L4-L9
+
+## Problems with world taxonomy
+
+The world taxonomy is not a well known part of GOV.UK and has a number of quirks that can make it problematic for developers working on GOV.UK. These quirks can produce a number of hard to diagnose problems.
+
+Any application that either tags content to taxons or pulls taxons out of the Publishing API has to consider them. Tagging to taxons requires an application to consider whether they will be accidentally overwriting any tagging to the world taxonomy. Pulling taxons from the Publishing API requires determining whether each item is part of the topic taxonomy or the world taxonomy.
+
+The rules regarding which departments and document types world taxons apply to are only enforced by Whitehall, not by content-tagger, meaning that these rules are not applied uniformly.
+
+There is a [world location][world-location] format that was the predecessor to the world taxon pages. This is still used for a number of things ([atom feed][atom-feed], tagging in Whitehall, rendering the [world page][world-page]) therefore we have two similar coupled concepts that are needed to be kept in sync.
+
+Content associated with a world location is tagged to the previous concept of a world location rather than a world taxon. For example [Information and events for UK nationals living in Portugal][portugal-info] has [no links][] to [Portugal][] and instead is linked to the now defunct Portugal world location. The consequence of this is that web links need to be [created based on the old world location concepts title][create-link] (which is a fragile process that needs manual overrides when it breaks) and that these content relationships cannot be mapped in data-science tooling without understanding this quirk.
+
+World taxons are treated the same as topic taxons in content tagger, the only clue a user has that they are dealing with something different is the /world part of their path.
+
+The structure of the world taxonomy is dense and repetitive. This results in a poor user experience for anyone tagging to it via Whitehall for example.
+
+When a user visits https://gov.uk/world the content they see is served from /world/all. This is because /world is rendered by whitehall and doesn't have its own content item.
```

Comment:
I think "served from /world/all" doesn't quite make sense as that makes it sound like a database. 

I did some learning about this (and discovered some surprises) and have this suggestion:

```
The [/world](https://gov.uk/world) page is unconventional for GOV.UK as it is not represented by a content item and it is instead only represented in router while being [rendered by Whitehall](https://github.com/alphagov/whitehall/blob/79515433a97f799a5f78f3410f5d598004bd91f2/config/routes.rb#L56). There is a content item at [/world/all](https://gov.uk/api/content/world/all) which redirects to /world, however the data from this content item is not used to render /world.
```

Reply:
So the content item for `/world/all` isn't used to render `/world`? What's it for?

Reply:
IKR 🤷‍♂ 

It might only exist to be a root of the taxonomy.

Reply:
This is odd :/ oh well, have updated :)